### DISCUSSION TITLE
Manual action cannot be triggered

### DISCUSSION THREAD

**Discussion by cschoreisz-pillar** [community/community#42166] [Actions] [answered]
### Select Topic Area

Bug

### Body

**Expected behaviour**
- Going to the Actions tab, I click on the workflow of my interest
- Click on the `Run workflow` button
- I should see a modal with the branch selector and run button

**Current behaviour**
- The modal is not displaying and is stuck on an endless loading state
- Error in the console: 
```
Request URL: https://github.com/withpillar/p/actions/manual?workflow=.github%252Fworkflows%252Fcd-staging.yaml
net::ERR_ABORTED 404
```
<img width="977" alt="Screen Shot 2022-12-19 at 6 31 45 PM" src="https://user-images.githubusercontent.com/114947427/208548258-09beb06b-3b4d-40d6-bebf-03755696a411.png">

**Context**
- I'm a collaborator of this project
- This was functional as of Friday (see screenshot, latest workflow run)
- I've tried in Incognito mode
- I'm using Chrome Version 108.0.5359.124 (Official Build) (x86_64)
- I'm on a Mac, macos Monterey (12.5.1)

**Comment by rcpeene** [comment, marked as answer]
Duplicate of [this submission!](https://github.com/community/community/discussions/42164)
